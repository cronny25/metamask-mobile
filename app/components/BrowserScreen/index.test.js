import React from 'react';
import { shallow } from 'enzyme';
import BrowserScreen from './';

describe('BrowserScreen', () => {
	it('should render correctly', () => {
		const wrapper = shallow(<BrowserScreen />);
		expect(wrapper).toMatchSnapshot();
	});
});
