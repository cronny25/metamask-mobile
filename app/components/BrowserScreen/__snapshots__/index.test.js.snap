// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BrowserScreen should render correctly 1`] = `
<Screen>
  <Browser
    defaultProtocol="https://"
    defaultURL="http://metamask.io"
  />
</Screen>
`;
